frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -292,27 +293,36 @@ export const PaymentInput = ({
 // Will be extended for stripe unconnected messages too
 const PaymentDisabledMessage = ({
   isEmailMode,
+  isStripeUnconnected,
 }: {
   isEmailMode: boolean
+  isStripeUnconnected: boolean
 }): JSX.Element | null => {
-  return isEmailMode ? (
+  return isEmailMode || isStripeUnconnected ? (
     <Box px="1.5rem" pt="2rem" pb="1.5rem">
       <InlineMessage variant={'info'}>
-        <Text>Payments are not available in email mode forms.</Text>
+        {isEmailMode ? (
+          <Text>Payments are not available in email mode forms.</Text>
+        ) : (
+          <Text>Connect your Stripe account in Settings.</Text>
+        )}
       </InlineMessage>
     </Box>
   ) : null
```

Comment:
This is kind of where I think having a sharded message component starts to gets a bit clunky. The logic for whether the `PaymentDisabledMessage` is displayed is actually identical to the `isDisabled` check, and so should be linked. Right now, there is no guarantee that if the toggle is disabled, we will definitely display a message. Instead, if we use the presence of a message to determine whether or not to disable the toggle, this would mean whether or not a message is present becomes the source of truth in whether the toggle is disabled. What do you think?

Reply:
I agree 😅 thanks Justyn [fb6ace7](https://github.com/opengovsg/FormSG/pull/6037/commits/fb6ace7ba0148a38493a86a34f43d7c608697bc7)